pkg/cli/cmd/start.go

```diff
@@ -131,3 +134,90 @@ func (cmd *StartCmd) env(rootPath string) map[string]string {
 		"CONTAINER_HOSTNAME": cmd.Hostname,
 	}
 }
+
+func getPorts(rootPath string) ([]string, error) {
+	portMap := make(map[string]string)
+	content, err := os.ReadFile(fmt.Sprintf("%s/cfg/config.yaml", rootPath))
+	if err != nil {
+		return nil, err
+	}
+
+	var api map[string]interface{}
+	err = yaml.Unmarshal(content, &api)
+	if err != nil {
+		return nil, err
+	}
+
+	healthConfig := getValueByKey(api["api"], "health")
```

Comment:
I would create a `getPort(any, string)` that takes the map value (`api["api"]` in this case) and returns the port.
Right now this logic is duplicated 4 times in this function.